docs/basic-tools.md

```diff
@@ -525,8 +525,8 @@ class ProofStream:
 ## Merkle Tree
 
 A [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree) is a vector commitment scheme built from a collision-resistant hash function[^4]. Specifically, it allows the user to commit to an array of $2^N$ items such that
```

Comment:
```suggestion
A [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree) is a vector commitment scheme built from a collision-resistant hash function[^4]. Specifically, it allows the user to commit to an array of $2^N$ items such that:
```
If the bullet items are standalone sentences, then I like to terminate the sentence that comes immediately before with some punctuation.